e2e/src/usecases/HandleDeepLinkSend.js

```diff
@@ -21,6 +17,12 @@ const launchDeepLink = async ({ url, newInstance = true }) => {
   })
 }
 
+const getFiatValue = async () => {
+  const sentAmountFiatElement = await element(by.id('SendAmountFiat')).getAttributes()
+  const sentAmountFiatValue = +sentAmountFiatElement.label.match(/(\d+\.\d+)/)[1]
```

Comment:
Can we document what this is parsing out, like an example full text and the portion we're extracting out.

Reply:
Simplified this in 2badc5b88a0be8bc105dcf4863152f9c1c04ff9b.